lib/portlayer/exec/config.go

```diff
@@ -38,9 +38,13 @@ type Configuration struct {
 
 	// Resource pools under which all containers will be created
 	ComputeResources []types.ManagedObjectReference `vic:"0.1" scope:"read-only"`
+
 	// Resource pool is the working version of the compute resource config
 	ResourcePool *object.ResourcePool
 
+	// uuid of portlayer vm
+	PortlayerUUID string
```

Comment:
This was added to help in filtering out the VCH appliance from the ps list.  I'm going to approach that a different way.  So it will be removed.